README.md

```diff
@@ -13,13 +14,46 @@ Running full scikit-learn test suite with daal4p's optimization patches
 - [![CircleCI](https://circleci.com/gh/IntelPython/daal4py.svg?style=svg)](https://circleci.com/gh/IntelPython/daal4py) when applied to scikit-learn from PyPi
 - [![CircleCI](https://circleci.com/gh/IntelPython/daal4py/tree/test-sklearn-master.svg?style=svg)](https://circleci.com/gh/IntelPython/daal4py/tree/test-sklearn-master) when applied to build from master branch
 
-With this daal4py API, your Python programs can use Intel(R) oneAPI Data Analytics Library algorithms in just one line:
+Core functioanlity of daal4py is in place Scikit-learn patching - Same Code, Same Behavior but faster execution. 
```

Comment:
I prefer to add info how to download daal4py, because currently it's only how to build daal4py.
# Installation
daal4py can be installed from conda:
```bash
conda install daal4py -c intel
```

Reply:
done